src/parser/jobs/mnk/modules/TwinSnakes.tsx

```diff
@@ -82,45 +88,33 @@ export default class TwinSnakes extends Module {
 			return
 		}
 
-		switch (action.id) {
-		// Ignore TS itself, plus Form Shift, Anatman, and Meditation
-		// Should we double count 6SS?
-		// We use gcdsSinceTS because we don't want to double count FPF
-		case (this.data.actions.TWIN_SNAKES.id):
-			if (this.twinSnake && !this.twinSnake.end && this.gcdsSinceTS < TWIN_SNAKES_CYCLE_LENGTH) {
-				this.earlySnakes++
-			}
+		// Ignore FS and Meditation
```

Comment:
Comment says form shift and meditation, but the skill list is Anatman and Meditation.  I'm assuming the skill list is wrong, since you're doing a check if Anatman extended twin snakes further down.

Reply:
Yes, good catch.